app/src/main/java/org/simple/clinic/editpatient/PatientEditScreenEvents.kt

```diff
@@ -1,6 +1,37 @@
 package org.simple.clinic.editpatient
 
+import org.simple.clinic.patient.Gender
 import org.simple.clinic.widgets.UiEvent
 import java.util.UUID
 
-data class PatientEditScreenCreated(val patientUuid: UUID): UiEvent
+data class PatientEditScreenCreated(val patientUuid: UUID) : UiEvent
+
+data class PatientEditPatientNameTextChanged(val name: String) : UiEvent {
+  override val analyticsName = "Patient Edit:Patient Name Text Changed"
```

Comment:
Can we use analytics keys similar to those in `PatientEntryScreen`? We're using `Create Patient Entry` there.